### PR TITLE
Get median

### LINKED ISSUE #8: Complete getMedian feature
- [x] Write a getMedian method that takes an array of integers as input, and returns the median of those integers as output.
- [x] Write a junit test case for the getMedian method.

### PR DESCRIPTION
**Pull request by aKrisnanto**
Added unit tests

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -24,8 +24,8 @@ public static float getMean(int[] arr) {
     public static float getMedian(int[] arr) {
     	Arrays.sort(arr);
     	if(arr.length%2 == 0) {//check if there are even amount of numbers
-			int mid = 1 + (int)(arr.length/2);
-			return (arr[mid]+arr[mid+1])/2;
+			int mid = arr.length/2;
+			return (arr[mid]+arr[mid-1])/2;
 			}
 		else {
 			int mid = (int)(arr.length/2);
```
Comment (JadenBalogh) [inline Main.java]: These still need a formatting pass to conform with our standards - you can use Alt+Shift+F in VS Code or follow the format of the existing methods to do this.

**Review by JadenBalogh — CHANGES_REQUESTED** [review]
Some formatting changes required. See comments.

**Review by JadenBalogh — APPROVED** [review]
Looks good! Feel free to merge.

**Comment by Jaredhw** [comment]
Approved - Merging into Master

**Pull request merged**